<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaText_Widget</class>
 <widget class="QWidget" name="MediaText_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Overlays</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <widget class="QGroupBox" name="add_text_groupbox">
     <property name="title">
      <string>Add Text Overlay</string>
     </property>
     <layout class="QVBoxLayout" name="add_text_layout">
      <item>
       <layout class="QHBoxLayout" name="text_input_layout">
        <item>
         <widget class="QLabel" name="text_label">
          <property name="text">
           <string>Text:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="text_input">
          <property name="placeholderText">
           <string>Enter text to display...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="orientation_layout">
        <item>
         <widget class="QLabel" name="orientation_label">
          <property name="text">
           <string>Orientation:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="orientation_combo">
          <item>
           <property name="text">
            <string>Horizontal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Vertical</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="orientation_spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="position_layout">
        <item>
         <widget class="QLabel" name="x_position_label">
          <property name="text">
           <string>X Position:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="x_position_spinbox">
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="y_position_label">
          <property name="text">
           <string>Y Position:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="y_position_spinbox">
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="color_font_layout">
        <item>
         <widget class="QLabel" name="color_label">
          <property name="text">
           <string>Color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="color_button">
          <property name="text">
           <string>Select Color</string>
          </property>
          <property name="styleSheet">
           <string>background-color: white; border: 1px solid black;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="font_size_label">
          <property name="text">
           <string>Font Size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="font_size_spinbox">
          <property name="minimum">
           <number>8</number>
          </property>
          <property name="maximum">
           <number>72</number>
          </property>
          <property name="value">
           <number>12</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="add_button_layout">
        <item>
         <spacer name="add_button_spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="add_text_button">
          <property name="text">
           <string>Add Text Overlay</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="add_button_spacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="overlays_groupbox">
     <property name="title">
      <string>Text Overlays</string>
     </property>
     <layout class="QVBoxLayout" name="overlays_layout">
      <item>
       <widget class="QLabel" name="overlays_info_label">
        <property name="text">
         <string>Right-click on entries for options. Double-click cells to edit.</string>
        </property>
        <property name="styleSheet">
         <string>color: gray; font-style: italic;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="overlays_table">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <column>
         <property name="text">
          <string>Enabled</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Text</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Orientation</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>X Position</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Y Position</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Color</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Font Size</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="overlay_buttons_layout">
        <item>
         <widget class="QPushButton" name="clear_all_button">
          <property name="text">
           <string>Clear All</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="overlay_buttons_spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="count_label">
          <property name="text">
           <string>Total: 0 overlays</string>
          </property>
          <property name="styleSheet">
           <string>color: gray;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="main_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 